romanesco/__init__.py

```diff
@@ -132,8 +131,7 @@ def convert(type, input, output, **kwargs):
         returns the output binding unchanged.
     """
 
-    if "data" not in input:
-        input["data"] = romanesco.io.fetch(input, **kwargs)
+    input["data"] = romanesco.io.fetch(input, **kwargs)
```

Comment:
Would it be the case that removing this short-circuit causes the same file to be downloaded twice, for example? Like when using `convert` and `validate`, for example?